### PR TITLE
Desktop version

### LINKED ISSUE #7: Create Navbar
### Create Navbar
- [ ] Responsive navbar for both `index.html` and `about.html`

### PR DESCRIPTION
**Pull request by didierganthier**
### Desktop Version and Javascript animations
In this pull request the following tasks were completed:
- [x] [Design `about.html` page desktop](https://github.com/didierganthier/event-website/commit/8712909741b59eb775f14d44e9710ad7fbf76bcb)
- [x] [Design program section](https://github.com/didierganthier/event-website/commit/dc12779874b6965088b40b86705de79337374365)
- [x] [Add dynamic speakers section using `index.js`](https://github.com/didierganthier/event-website/commit/6cec51f177687287c4c27ea2159acd25b3de1291)
- [x] [Add dynamic colors](https://github.com/didierganthier/event-website/commit/b0d9a589f81b36b5910d9d092d7d76539b3fb917)
- [x] [Add javascript animations using `index.js` and `about.js`](https://github.com/didierganthier/event-website/commit/26f110578b6a6c826a6717d1a310eb8f92b924dc)
- [x] [Add Lato font](https://github.com/didierganthier/event-website/commit/09c743f211668e5e0339693e4ce6d02d030cad5e)
- [x] [Add formspree](https://github.com/didierganthier/event-website/pull/14/commits/0f14ba06367928b8e43b25a71cede38deba39c85)

### After Review
- [x] [Fix relative links issues](https://github.com/didierganthier/event-website/pull/14/commits/e7cf10a4a56e29e8f1f2c630e268907d198f7a19)
- [x] [Fix background issues](https://github.com/didierganthier/event-website/pull/14/commits/446a5f56506ab49ceb4f4320e4f836e00a3d218c)
- [x] [Fix images paths](https://github.com/didierganthier/event-website/pull/14/commits/70ec2029195dff85a5e7303c61066888730e4ff1)
- [x] [Deploy to `Github Pages`](https://didierganthier.github.io/event-website/about.html)

To build Tailwind CLI:
`npx tailwindcss -i ./styles.css -o ./dist/output.css --watch`

Find below the recording of the presentation:
[Loom Recording of the presentation](https://www.loom.com/share/b94f799501be42f1b8d6d9c73ffbaa1b)

### CODE REVIEW

**Review by franclobo — COMMENTED** [review]
Hi @didierganthier , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that appears in my dashboard as Day 3 of the Capstone project, nevertheless, I confirmed with you and you said that has submitted it the day 2.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by m05e5 — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @didierganthier  , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Hi. You are required to add an `Acknowledgements` section to your readme and appropriate credit to **Cindy Shin** the author of the original design.
- [ ] Although Versel is a great tool for hosting apps, You are required to use GitHub page for this project. So please kindly deploy your app using GitHub page
- [ ] Please as recommended try to stick to the design as much as possible so, kindly set the footer background of the about page to white and the footer of your Home page to dark as shown in the design template 
**About page**
image

**Home Page**
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_m05e5_


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Event Website to announce upcoming NFT Haïti Conference
 
 ## Live Demo
 
-- N/A
+- [Live Website](https://nftokap.vercel.app/)
```
Comment (danmainah) [inline README.md:14]: - [ ] [optional] Kindly change the link of your live version to the GitHub live version.

**Review by danmainah — APPROVED** [review]
Hi @didierganthier , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

- [ ] It would be nice if your footer content were the same for both index and about page.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**